docs/topics/cli-run.adoc

```diff
@@ -11,11 +11,11 @@ You can run {ProductShortName} against your application.
 .Procedure
 
 . Open a terminal and navigate to the `<{ProductShortName}_HOME>/bin/` directory.
-. Execute the `windup-cli` script, or `windup-cli.bat` for Windows, and specify the appropriate arguments:
+. Execute the `{mta-cli}` script, or `{mta-cli}.bat` for Windows, and specify the appropriate arguments:
```

Comment:
mta-cli for windows is not a `.bat` anymore, now it is a `.exe` file

Reply:
@abrugaro  does this apply to all mentions of batch files that I find in the documentation? 

I found these instances: 
NOTE: In a Windows environment, use the `run_mta.bat` script.

NOTE: In a Windows environment, use the `switch_to_authentication_required.bat` script.

C:\<{ProductShortName}_HOME>\run_windup.bat   (should this also say run_mta.exe???)

Execute the `{LC_PSN}-cli` script, or `{LC_PSN}-cli.bat` for Windows, and specify the appropriate arguments:

> ...\bin\standalone.bat -Djboss.socket.binding.port-offset=<offset_value>


C:\<MTR_HOME>\switch_to_authentication_required.bat

C:\<MTR_HOME>\run_windup.bat

C:\<MTR_HOME>\switch_to_automatic_authentication.bat